homeassistant/components/squeezebox/media_player.py

```diff
@@ -469,15 +470,11 @@ async def async_turn_on(self):
         await self._player.async_set_power(True)
 
     async def async_play_media(self, media_type, media_id, **kwargs):
-        """
-        Send the play_media command to the media player.
-
-        If ATTR_MEDIA_ENQUEUE is True, add `media_id` to the current playlist.
-        """
+        """Send the play_media command to the media player."""
         cmd = "play"
         index = None
 
-        if kwargs.get(ATTR_MEDIA_ENQUEUE):
+        if kwargs.get(ATTR_MEDIA_ENQUEUE) == MediaPlayerEnqueue.ADD:
             cmd = "add"
 
```

Comment:
Squeezebox supports "next", which we can implement as follows
```suggestion
        if kwargs.get(ATTR_MEDIA_ENQUEUE) == MediaPlayerEnqueue.NEXT:
            cmd = "insert"
```

Reply:
Thanks. Pushed.